### ISSUE TITLE
improving fine-tuning quality

### ISSUE THREAD

**Issue by harshaUwm163** [rinongal/textual_inversion#126]
Hello, Thank you for this great project and the code! 
I have a small set of birds images that I want to fine-tune on, They look like this:
<img src="https://user-images.githubusercontent.com/88546131/208314268-1a8ae47d-f008-4c0e-91d6-3a9ee47e8f16.jpg" height="150" />  <img src="https://user-images.githubusercontent.com/88546131/208314475-d82f1232-e32a-4b69-bed3-6d8456488998.jpg" height="150" />  <img src="https://user-images.githubusercontent.com/88546131/208314511-7477497a-4619-4f33-82b1-af78e9579c55.jpg" height="150" />   <img src="https://user-images.githubusercontent.com/88546131/208314534-a78ce898-0f31-45da-a1e9-d2f8f2eab9bf.jpg" height="150" /> 

I am using '*' as my placeholder string. After 6100 training iterations, the images generated look something like this:
<img src="https://user-images.githubusercontent.com/88546131/208314711-f52d9715-3214-49e2-80d5-1e669d556395.jpg" height="150" />  <img src="https://user-images.githubusercontent.com/88546131/208314755-5edfa6ec-bd6f-4f4a-82af-ee94f69a3469.jpg" height="150" />  <img src="https://user-images.githubusercontent.com/88546131/208314774-9d64f19c-f552-4488-8e94-29aa5b16152c.jpg" height="150" />   <img src="https://user-images.githubusercontent.com/88546131/208314787-7fa3f857-6a87-46ff-8612-e24c5fde3548.jpg" height="150" /> 

These are pretty good, but is there a way to improve the quality of the final images?
I am using the large ldm model and all the default parameters.

Thanks!

**Comment by rinongal** [comment]
That depends on what quality improvement you're looking for. If you're talking about better resolutions and shapes, you might have better results with Stable Diffusion, but editability suffers there.

If you want it to capture finer details and you don't care about editing, you can just bump up the number of learned vectors (num_vectors_per_word) or increase the learning rate.

**Comment by harshaUwm163** [comment]
Thank you @rinongal  for the detailed explanation. 

What do you mean by "editability" with Stable Diffusion here? (Sorry if it is obvious, but I am new to Diffusion Models).

**Comment by rinongal** [comment]
By editability I mean how well you can change the object with your prompts.
If you train your concept too strongly, then things like "a photo of *" would give your birds, but things like "a painting of * in the style of Van Gogh" will still give photo-realistic images, and not paintings in the style of Van Gogh.

If all you care about is generating more of your subjects, then you don't care about their editability.

**Comment by harshaUwm163** [comment]
Gotcha! Thank you for the reply! I am closing this issue now!
